practice/00-HTML-CSS-basics/HelloWorld.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+    <head>
+		<title>Hello World!</title>
+	</head>
```

Comment:
Please check that the code is properly indented. Also the idea of this commentary is to do a modification to just approve, testing as appropriate when there is an error or something changed. With this you will learn the full flow.